pkgs/applications/office/libreoffice/default-gen-shell.nix

```diff
@@ -19,9 +19,11 @@ stdenv.mkDerivation {
     builder = ./download-list-builder.sh;
   };
 
+  buildInputs = [ python36 ];
```

Comment:
Ah, I didn't know 3.6 was considered pre-release. I think I just wanted string interpolation. Guess we can do without it.